lib/fizzy/parser.cpp

```diff
@@ -259,6 +277,9 @@ Module parse(bytes_view input)
         if (it != expected_end_pos)
             throw parser_error{"incorrect section " + std::to_string(static_cast<int>(id)) +
                                " size, difference: " + std::to_string(it - expected_end_pos)};
+
+        if (module.tablesec.size() > 1)
```

Comment:
I have another set of changes which updates the parser for the memory section, etc.